game/game.go

```diff
@@ -645,4 +645,16 @@ var taunts = []string{
 	"I think you should resign.",
 	"This just isn't your game.",
 	"Good heavens, are you still trying to win?",
-	"Stop! He's already dead!"}
+	"Stop! He's already dead!",
+	"Instead of watching this, I think I'll go find a lake of piranhas to jump in.",
+	"Ban Chi is _supposed_ to be fun.",
```

Comment:
You do realize adding underscores makes you responsible for implementing markdown in chat. #68